scripts/release_issue_status/main.py

```diff
@@ -79,6 +83,14 @@ def _extract_author_latest_comment(comments):
     return _NULL if not q else q[-1][1]
 
 
+def _whether_author_comment(comments):
+    q = set(comment.user.login for comment in comments)
+    for administrators in _PYTHON_SDK_ADMINISTRATORS:
+        q.discard(administrators)
+
+    return True if len(q) > 0 else False
```

Comment:
the code could be cleaner ：https://blog.csdn.net/isoleo/article/details/13000975
![image](https://user-images.githubusercontent.com/70930885/128312875-2b7b21e3-217a-48a9-9aae-9cb9ef6fc13a.png)